app/models/feedback.rb

```diff
@@ -0,0 +1,4 @@
+class Feedback < ApplicationRecord
+    validates :rating, presence: true
+    belongs_to :feedbackable, polymorphic: true
```

Comment:
Is it better to make this polymorphic rather than just having two optional relationships to resources and services? I guess there's some advantage in that you won't orphan an object with a relationship to nothing but it might make sql queries more difficult. I'm fine with it either way though. Also as you mentioned to me, if you remove the old tables then you could at least use a more sensible name "reviewable" rather than "feedbackable."